*** CONTRIBUTING.md	2017-01-08 23:55:18.230690813 -0800
--- CONTRIBUTING.md__2017-01-09--00.12.58	2017-01-09 00:12:58.463986619 -0800
***************
*** 16,24 ****
  * [Windows Printing](https://github.com/thinkle/gourmet/issues/708).
  * [Release a mac version](https://github.com/thinkle/gourmet/issues/704).
  * Code cleanup.
!  * This includes using [SQLAlchemy's ORM](https://github.com/thinkle/gourmet/issues/712), which is an ongoing effort in the `develop` branch, and scheduled for release woth version 0.18.0.
   * Parts of Gourmet's source code are about ten years old, and technologies have evolved since development started. Hence, there are some things that were implemented differently back then and could be greatly simplified by using modern technologies available in the present-day versions of the toolkits used by Gourmet (such as Gtk components as in [gourmet/gtk_extras/dialog_extras.py](../blob/master/gourmet/gtk_extras/dialog_extras.py), database backends, etc).
!  * Also, there are still large portions of commented-out code where something already got replaced by something else, 
   * or python modules that aren't actually imported anywhere else anymore (example: [3daaba2db](../commit/3daaba2dbb865272cb5b1b96d14eca3fe5deeaf7)).
   * Remove old logo images, file duplicates, obsolete plugins, etc.
   * Clean up files names which currently are a mixture of camelCaps and under\_scores, plus some rather obscure abbreviations.
--- 16,24 ----
  * [Windows Printing](https://github.com/thinkle/gourmet/issues/708).
  * [Release a mac version](https://github.com/thinkle/gourmet/issues/704).
  * Code cleanup.
!  * This includes using [SQLAlchemy's ORM](https://github.com/thinkle/gourmet/issues/712), which is an ongoing effort in the `develop` branch, and scheduled for release with version 0.18.0.
   * Parts of Gourmet's source code are about ten years old, and technologies have evolved since development started. Hence, there are some things that were implemented differently back then and could be greatly simplified by using modern technologies available in the present-day versions of the toolkits used by Gourmet (such as Gtk components as in [gourmet/gtk_extras/dialog_extras.py](../blob/master/gourmet/gtk_extras/dialog_extras.py), database backends, etc).
!  * Also, there are still large portions of commented-out code where something already got replaced by something else,
   * or python modules that aren't actually imported anywhere else anymore (example: [3daaba2db](../commit/3daaba2dbb865272cb5b1b96d14eca3fe5deeaf7)).
   * Remove old logo images, file duplicates, obsolete plugins, etc.
   * Clean up files names which currently are a mixture of camelCaps and under\_scores, plus some rather obscure abbreviations.
